docs-next/pages/roadmap.mdx

```diff
@@ -0,0 +1,121 @@
+import { Markdown } from '../components/Page';
+
+# Roadmap
+
+> Note: this page is WIP, and will be updated based on research and feedback
+
+When Keystone 5 was announced in 2018, it was a great leap forward for the project from a back-end
+Node.js CMS platform built with MongoDB, to a modern platform with first class support for GraphQL,
+PostgreSQL, and more.
+
+Three years on, Keystone Next is a preview of the next major release of KeystoneJS, the most
+powerful headless content management system around. We've substantially rewritten the CLI, Schema
+config, and Admin UI to make them more powerful and easier to use than ever before.
+
+We've also created a new Document field type that lets you compose content from customisable
+component blocks, making Keystone the first headless CMS built for the Design System generation.
+
+#### Here's what's next on our list:
```

Comment:
Heading level `####` looks really weird here 🤔

Reply:
Yeah, fwiw I played with a few ways of doing this and landed on this one looks better than the rest (did you review the preview, or just the markdown?)

I'd like to leave it like this now for effect, and we'll get a design pass on it when we've got more of the content in a few days

Reply:
I reviewed the preview, IMHO it looks like a mistake.

Reply:
Now I think about it, the reason I noticed this, but not the equivalent heading in the what's new doc is that the what's new doc has an emoji next to it. I think something similar here would make it looks less out of place.

Reply:
Haha I *had* an emoji on it while I was writing, and thought it looked too cute. Will revert, hang on!